flash/core/utilities/imports.py

```diff
@@ -301,7 +301,9 @@ def _import_module(self):
     topic = os.environ["FLASH_TEST_TOPIC"]
     _CORE_TESTING = topic == "core"
     _IMAGE_TESTING = topic == "image"
-    _IMAGE_EXTRAS_TESTING = topic == "image,image_extras" or topic == "icevision" or topic == "vissl"
+    _IMAGE_EXTRAS_TESTING = (
+        topic == "image,image_extras,image_extras_effdet" or topic == "icevision" or topic == "vissl"
+    )
```

Comment:
This seems not quite right. We still need to check for `topic == "image,image_extras"` or the regular image extras tests won't run right?

Reply:
Reverted this changed based on [our conversation here](https://github.com/Lightning-AI/lightning-flash/pull/1410#discussion_r944441908).